calblink.go

```diff
@@ -73,6 +73,39 @@ import (
 // MultiEvent indicates whether to show two events if there are multiple events in the time range.
 
 // responseState is an enumerated list of event response states, used to control which events will activate the blink(1).
+
+type ClientCredentials struct {
```

Comment:
We don't need to decompose the credentials, they're fine as an opaque object.  This struct isn't needed.